Public/Src/Cache/DistributedCache.Host/Configuration/DistributedContentSettings.cs

```diff
@@ -177,6 +177,30 @@ private DistributedContentSettings(IDictionary<string, string> connectionSecretN
         [DataMember]
         public int EvictionWindowSize { get; set; } = 500;
 
+        /// <summary>
+        /// Delays for retries for file copies
+        /// </summary>
+        [DataMember]
+        public int[] RetryIntervalForCopiesMs { get; set; } =

```

Comment:
Ignorant question here: is it possible for serialization to actually set this to null? if so, things would explode both in the line below and in the code that uses it. There is no `Contract.Requires` in http://index/?leftProject=BuildXL.Cache.ContentStore.Distributed&leftSymbol=bjztwptambty&file=Stores%5CDistributedContentCopier.cs&line=60

Reply:
Not ignorant at all. It could happen. Now that two of you have pointed this out, I'll add some handrails.

---
In reply to: [307927695](https://github.com/microsoft/BuildXL/pull/667#discussion_r307927695) [](ancestors = 307927695)